policyengine_canada/variables/gov/provinces/qc/tax/income/credits/solidarity/qc_solidarity_eligibility.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_canada.model_api import *
+
+
+class qc_solidarity_eligibility(Variable):
+    value_type = bool
+    entity = Household
+    label = "Quebec solidarity tax credit eligibility"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.qc.tax.income.credits.solidarity.eligibility
+
+        income = household("adjusted_family_net_income", period)
+
+        person = household.members
+        # Tax filer eligibility
+        # You were 18 or older
+        age_eligible = person("age", period) >= p.age
+        # You were younger than 18 and met all following requirements
+        has_spouse = household("is_married", period)
+        children = person("own_children_in_household", period)
+        has_child = children > 0
+        emancipated = person("is_emancipated", period)
+
+        eligible = age_eligible | (has_spouse & has_child & emancipated)
+
+        # family income eligibility
```

Comment:
Also, could you help take another look at this family income eligibility? @MaxGhenis 

The [Revenue Quebec](https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/how-to-complete-your-income-tax-return/line-by-line-help/solidarity-tax-credit/) website of calculation the reduction of solidarity credit, it also mentioned that "If your principal residence is located in a northern village, do not take the table into account" which corresponds to the maximum family income table. 

Should I assume that household income is not taken into account for the credit eligibility, as long as one lives in a northern village?

Reply:
@MaxGhenis friendly reminder